CorpusCallosum/fastsurfer_cc.py

```diff
@@ -0,0 +1,856 @@
+import argparse
+import json
+from pathlib import Path
+
+# import warnings warnings.filterwarnings("ignore", message="TypedStorage is deprecated")
+import nibabel as nib
+import numpy as np
+import torch
+
+import FastSurferCNN.utils.logging as logging
+from CorpusCallosum.data.constants import (
+    CC_LABEL,
+    FSAVERAGE_CENTROIDS_PATH,
+    FSAVERAGE_DATA_PATH,
+    FSAVERAGE_MIDDLE,
+    STANDARD_OUTPUT_PATHS,
+    WEIGHTS_PATH,
+)
+from CorpusCallosum.data.read_write import (
+    convert_numpy_to_json_serializable,
+    get_centroids_from_nib,
+    load_fsaverage_centroids,
+    load_fsaverage_data,
+    run_in_background,
+    save_nifti_background,
+)
+from CorpusCallosum.localization import localization_inference
+from CorpusCallosum.registration.mapping_helpers import (
+    apply_transform_and_map_volume,
+    apply_transform_to_pt,
+    get_mapping_to_standard_space,
+    interpolate_midplane,
+    map_softlabels_to_orig,
+)
+from CorpusCallosum.segmentation import segmentation_inference, segmentation_postprocessing
+from CorpusCallosum.shape.cc_postprocessing import (
+    check_area_changes,
+    create_visualization,
+    make_subdivision_mask,
+    process_slices,
+)
+from FastSurferCNN.data_loader.conform import is_conform
+from recon_surf import lta
+from recon_surf.align_points import find_rigid
+
+logger = logging.get_logger(__name__)
+
+
+def options_parse() -> argparse.Namespace:
+    """Parse command line arguments for the pipeline."""
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--in_mri",
+        type=str,
+        required=False,
+        help="Input MRI file path. If not provided, defaults to subject_dir/mri/orig.mgz",
+    )
+    parser.add_argument(
+        "--cpu",
+        action="store_true",
+        help="Force CPU usage even when CUDA is available",
+    )
+    parser.add_argument(
+        "--aseg",
+        type=str,
+        required=False,
+        help="Input segmentation file path. If not provided, defaults to subject_dir/mri/aparc.DKTatlas+aseg.deep.mgz",
+    )
+    parser.add_argument(
+        "--subject_dir",
+        type=str,
+        required=False,
+        help="Subject directory containing standard FreeSurfer structure. "
+        "Required if --in_mri and --aseg are not both provided.",
+        default=None,
+    )
+    parser.add_argument("--debug_output_dir", type=str, required=False, default=None,
+                        help="Directory for debug output (default: subject_dir/qc_snapshots)")
```

Comment:
```suggestion
    parser.add_argument(
        "--qc_snapshots",
        dest="qc_snapshots",
        type=Path, 
        required=False,
        default=None,
        help="Directory to store snapshots for QC in (default: do not store)",
    )
    parser.add_argument(
        "--qc_snaps",
        action="store_const",
        const="qc_snapshots",
        dest="qc_snapshots",
        help="Shortcut to store snapshots for QC in `subject_dir/qc_snapshots` (== `--qc_snapshots qc_snapshots`)",
    )
```

Make it consistent with HypVINN.